packages/extension-browser/src/devtools/views/pages/config/resources.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import { FormEvent } from 'react';
+import escapeRegExp = require('lodash/escapeRegExp');
+
+import { browser } from '../../../../shared/globals';
+
+import { getMessage } from '../../../utils/i18n';
+
+import ConfigExample from './config-example';
+import ConfigLabel from './config-label';
+import ConfigSection from './config-section';
+import ValidInput from './valid-input';
+
+const enum Ignore {
+    ThirdParty = '--webhint-third-party'
+}
+
+type Props = {
+    query?: string;
+    onChange: (query?: string) => void;
+};
+
+const placeholder = 'google-analytics\.com';
+
+/** Create a regular expression to exclude URLs not part of the current origin. */
+const buildIgnoreThirdParty = (): Promise<string> => {
+    return new Promise((resolve) => {
+        browser.devtools.inspectedWindow.eval('location.origin', (origin: string) => {
+            resolve(`^(?!${escapeRegExp(origin)})`);
+        });
+    });
+};
+
+/** Check if a user's custom `browserslist` query is valid, notifying them if it is not. */
```

Comment:
This comment talks about browserslist but we are verifying the regex for ignored resources, right?

Reply:
Yep. 🤦‍♂️